### PR TITLE
AtomicHabit Feature  checkstyle fixes

### PR DESCRIPTION
**Pull request by YongbinWang**
Fixed allocated check style issues.

### CODE REVIEW

**Review by nichyjt — APPROVED** [review]
LGTM!

File: src/main/java/wellnus/atomichabit/feature/AtomicHabit.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 package wellnus.atomichabit.feature;
 
+/**
+ * Class to represent a unique atomic habit that the user will practice
+ * It contains primarily the description of the habit and the count of the habit
```
Comment (haoyangw) [inline src/main/java/wellnus/atomichabit/feature/AtomicHabit.java:5]: Remember to add '.' at the end of each sentence!

File: src/main/java/wellnus/atomichabit/feature/AtomicHabitList.java [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 import java.util.ArrayList;
 
+/**
+ * Class to represent a container that will contain all unique AtomicHabit objects in an arraylist
```
Comment (haoyangw) [inline src/main/java/wellnus/atomichabit/feature/AtomicHabitList.java:6]: Remember to add '.' at the end of the sentence!

File: src/main/java/wellnus/atomichabit/feature/AtomicHabitManager.java [hunk 3]
```diff
@@ -13,14 +13,18 @@
 import wellnus.manager.Manager;
 import wellnus.ui.TextUi;
 
+/**
+ * Class to represent the event driver of Atomic Habits feature
+ * This class will handle calling the different available commands for Atomic Habits according to user input
```
Comment (haoyangw) [inline src/main/java/wellnus/atomichabit/feature/AtomicHabitManager.java:18]: Remember to add '.' at the end of each sentence!

File: src/main/java/wellnus/atomichabit/feature/AtomicHabitManager.java [hunk 4]
```diff
@@ -34,6 +38,10 @@ public class AtomicHabitManager extends Manager {
     private final TextUi textUi;
     private final AtomicHabitList habitList;
 
+    /**
+     * Constructor of AtomicHabitManager
+     * Will initialise the private objects habitList and textUi
```
Comment (haoyangw) [inline src/main/java/wellnus/atomichabit/feature/AtomicHabitManager.java:43]: Remember to add '.' at the end of each sentence!

File: src/test/java/wellnus/reflection/HomeCommandTest.java [hunk 5]
```diff
@@ -1,13 +1,18 @@
...
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.Test;
+
+import wellnus.exception.BadCommandException;
 
+/**
+ * Class to test different tests for HomeCommand Class utilising JUnit tests
+ * Test cases will involve expected outputs and correct exception handling
```
Comment (haoyangw) [inline src/test/java/wellnus/reflection/HomeCommandTest.java:14]: Remember to add '.' at the end of each sentence!

**Review by haoyangw — CHANGES_REQUESTED** [review]
Good fixes for the checkstyle errors! I'll need you to add the missing fullstops though, then it'll be LGTM!

**Pull request merged**